### DISCUSSION TITLE
QuestCraft v2.0.0

### DISCUSSION THREAD

**Discussion by gusbunce** [QuestCraftPlusPlus/QuestCraft#371] [General]
1. Added a Modmanager (Thanks MrNavaStar)
2. Added immersive controls (Punch blocks and stuffs)
3. Added haptics
4. Teleportation 
5. SVC (SimpleVoiceChat) compat
6. Added support for Minecraft 1.19 and Minecraft 1.18 dual-install
7. The toast is disabled by default in options.txt
8. Modmanager has auto updates built-in
9. JNA is now initialized by default
10. Fabric is now the only version shown by default
11. Updated Holy Gl4ES
12. Fixed using system EGL for creating the pbuffer surface
13. Fixed QuickChat issue
14. More performance by canceling desktop rendering
15. Screen overlays

<hr /><em>This discussion was created from the release <a href='https://github.com/QuestCraftPlusPlus/QuestCraft/releases/tag/'>QuestCraft v2.0.0</a>.</em>

**Comment by LameMan0** [comment]
Ok

**Reply by BruhBruh590** [reply]
sound not working......

**Comment by wellsilver** [comment]
Nice
